### ISSUE TITLE
Use BS fullscreen message

### ISSUE THREAD

**Issue by Swiip** [FountainJS/generator-fountain-browsersync#2]
I think it would be great to use this new BS plugin.

https://github.com/shakyShane/bs-fullscreen-message

**Comment by zckrs** [comment]
IMO this issue is already fixed

**Comment by Swiip** [comment]
Yep, I discovered we already have this with the webpack middleware. But I think it's only the case with React. Other confs doesn't (yet?) use the middleware.
